manon/pagination.scss

```diff
@@ -92,7 +92,6 @@
   gap: var(--pagination-list-gap);
 
   li {
-    margin-top: -0.1rem; /* fix for render issue; white space between this li and the parent ul border */
```

Comment:
This property is overridden by the `margin: 0` on line 101. Removing it seems to have no effect, at least in Firefox. @HeleenSG do you know if this fix is still needed?

Reply:
For future hacks, lets include the original issue number in the comment line (and also produce a testcase referencing that issue)